### PR TITLE
chore: add volunteer abhijith-zero

### PR DESCRIPTION
**Pull request by abhijith-zero**

### CODE REVIEW

File: data/users/AbhijithRajesh.json [hunk 1]
```diff
@@ -0,0 +1,18 @@
+{
```
Comment (codewithvoid) [inline data/users/AbhijithRajesh.json:1]: Rename filename to `abhijith-zero.json`

File: data/users/AbhijithRajesh.json [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+  "name": "Abhijith Rajesh",
+  "bio": "Full Stack Engineer",
+  "image": "https://avatars.githubusercontent.com/u/65908163?v=4",
+  "categories": [
+    {
+      "slug": "web-development",
+      "topics": ["VueJs", "Laravel", "NestJs","react"]
```
Comment (codewithvoid) [inline data/users/AbhijithRajesh.json:8]: Limit these to max 2-3 topics

File: data/users/AbhijithRajesh.json [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+    },
+    {
+      "slug": "programming",
+      "topics": ["Python", "PHP", "javascript","C"]
```
Comment (codewithvoid) [inline data/users/AbhijithRajesh.json:12]: Same here. Max 2-3.

**Review by codewithvoid — CHANGES_REQUESTED** [review]
Thanks for volunteering @abhijith-zero. Left some comments.

File: data/users/AbhijithRajesh.json (hunk 2 above)
Comment (abhijith-zero) [reply data/users/AbhijithRajesh.json]: Done

File: data/users/AbhijithRajesh.json (hunk 3 above)
Comment (abhijith-zero) [reply data/users/AbhijithRajesh.json]: Done

**Review by codewithvoid — APPROVED** [review]
Thanks for making the changes. Welcome!

**Pull request merged**